console-csharp-connect-sample/Constants.cs

```diff
@@ -1,18 +1,14 @@
 ﻿//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
 //See LICENSE in the project root for license information.
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
 namespace console_csharp_connect_sample
 {
     internal class Constants
     {
-        public const string ClientId = "ENTER_YOUR_CLIENT_ID";
+		public const string ClientId = ""; // "ENTER_YOUR_CLIENT_ID";
+		public const string Authority = "https://login.microsoftonline.com/common"; // "ENTER_THE_SIGN_IN_AUTHORITY"
```

Comment:
this should also be in a config file so that your sample also works in sovereign and national clouds. The Graph base URL should be as well.

Reply:
Noted. Will move these to a config file